utils/docker-ce/files/dockerd.init

```diff
@@ -3,12 +3,83 @@
 USE_PROCD=1
 START=25
 
+DOCKERD_CONF="/tmp/dockerd/daemon.json"
+
+json_add_array_string() {
+	json_add_string "" "$1"
+}
+
+process_config() {
+	local alt_config_file data_root log_level
+
+	[ -f /etc/config/dockerd ] || return 0
+
+	rm -f "$DOCKERD_CONF"
+	config_load 'dockerd'
+
+	config_get alt_config_file globals alt_config_file
```

Comment:
This might get reworked depending on how you want to address the previous comments, so FYI, this looks like it should have been
```suggestion
	config_get alt_config_file globals ""
```